### PR TITLE
Create a database based on a diagram

### PR DESCRIPTION
**Pull request by Rudy-Menekam**
- Create a file named schema_based_on_diagram.sql
- Create tables for patients, treatment, invoices, invoice_items, and medical_histories

### CODE REVIEW

**Review by Davidosky007 — COMMENTED** [review]
Hi @Rudy-Menekam @alicemirigo92 👋 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that doesn't show code contribution from your partner as this is a paired programming activity you are expected to work together with your partner on this project 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: schema_based_on_diagram.sql [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ADD FOREIGN KEY (invoice_id) REFERENCES invoices (id);
+
+ALTER TABLE invoice_items 
+ADD FOREIGN KEY (treatment_id) REFERENCES treatments (id);
+
+ALTER TABLE invoices 
+ADD FOREIGN KEY (medical_history_id) REFERENCES medical_histories (id);
+
+ALTER TABLE medical_histories 
+ADD FOREIGN KEY (patient_id) REFERENCES patients (id);
```
Comment (zieeco) [inline schema_based_on_diagram.sql]: - Hi :clap: , Let's not forget to add the indexes on all the foreign keys. see an example: 
- kindly do the same for every other foreign key to meet the project requirement.

```sql
CREATE INDEX ON invoice_items (invoice_id);
CREATE INDEX ON invoice_items (treatment_id);
```

File: schema_based_on_diagram.sql [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    date_of_birth DATE NOT NULL
+);
+
+CREATE TABLE medical_histories (
+    id SERIAL PRIMARY KEY,
+    admitted_at timestamp NOT NULL,
+    patient_id int NOT NULL,
+    status varchar(50) NOT NULL
+);
+
```
Comment (zieeco) [inline schema_based_on_diagram.sql:13]: - Let's also create the  "join table" that establishes the many-to-many relationships. Kindly create a table between `medical_histories` and `treatments` to establish that. e.g:



```sql
CREATE TABLE medical_histories_has_treatments (
    medical_history_id int references medical_histories(id),
    treatment_id int references treatments(id),
    );
```

**Review by zieeco — CHANGES_REQUESTED** [review]
Hi @Team,





Great job so far! 👏 
You've done well, however, there are some issues that you still need to work on to go to the next project, but you are almost there!


## To highlight 🎯

- No linter errors.✔️
- Nice schema file. ✔️
- Descriptive PR. ✔️
- Good commit messages :heavy_check_mark: 

## Required Changes ♻️
_Check the comments under the review._




## Optional suggestions
- N/A


**_Every comment with the **[OPTIONAL]** prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._**


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, @Rudy-Menekam.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- Fixed all suggestions from the reviewer 👍 

### Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**